src/dom.js

```diff
@@ -124,7 +124,7 @@ function scrollIntoView(el: HTMLElement, container: any): void {
   }
 
   const bbox = el.getBoundingClientRect();
-  container.scrollTo({ top: bbox.top + containerTop - (containerHeight - bbox.height) / 2 });
+  container.scrollTo(window.scrollX, bbox.top + containerTop - (containerHeight - bbox.height) / 2);
```

Comment:
For my own reference, this is one of the two options here: https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo